roofit/roofitcore/inc/RooGenericPdf.h

```diff
@@ -25,6 +25,7 @@ class RooArgList ;
 class RooGenericPdf : public RooAbsPdf {
 public:
   // Constructors, assignment etc
+  inline RooGenericPdf(){}
```

Comment:
maybe:
```suggestion
  RooGenericPdf() = default;
```
[Note that the inline keyword is already implied if the definition is inside the class declaration]